test/ejabberd_tests/tests/ejabberdctl_SUITE.erl

```diff
@@ -956,19 +1015,31 @@ remove_expired_messages_test(Config) ->
         JidB = nick_to_jid(kate, Config),
         JidRecordMike = rpc_call(jid, from_binary, [JidA]),
         JidRecordKate = rpc_call(jid, from_binary, [JidB]),
-        Msg1 = escalus_stanza:chat_to(<<"kate@localhost">>, "Rolling stones"),
-        Msg2 = escalus_stanza:chat_to(<<"kate@localhost">>, "Arctic monkeys!"),
-        Msg3 = escalus_stanza:chat_to(<<"kate@localhost">>, "More wine..."),
-        Msg4 = escalus_stanza:chat_to(<<"kate@localhost">>, "kings of leon"),
+        Domain = domain(),
+        Msg1 = escalus_stanza:chat_to(<<"kate@", Domain/binary>>,
+                                      "Rolling stones"),
+        Msg2 = escalus_stanza:chat_to(<<"kate@", Domain/binary>>,
+                                      "Arctic monkeys!"),
+        Msg3 = escalus_stanza:chat_to(<<"kate@", Domain/binary>>,
+                                      "More wine..."),
+        Msg4 = escalus_stanza:chat_to(<<"kate@", Domain/binary>>,
+                                      "kings of leon"),
         OldTimestamp = fallback_timestamp(10, now()),
         ExpirationTime = fallback_timestamp(2, now()),
         ExpirationTimeFuture= fallback_timestamp(-5, now()),
-        OfflineOld = generate_offline_expired_message(JidRecordMike, JidRecordKate, Msg1, OldTimestamp, ExpirationTime),
-        OfflineNow = generate_offline_expired_message(JidRecordMike, JidRecordKate, Msg2, now(), ExpirationTime),
-        OfflineFuture = generate_offline_expired_message(JidRecordMike, JidRecordKate, Msg3, now(), ExpirationTimeFuture),
-        OfflineFuture2 = generate_offline_expired_message(JidRecordMike, JidRecordKate, Msg4, OldTimestamp, ExpirationTimeFuture),
+        OfflineOld = generate_offline_expired_message(JidRecordMike,
+                             JidRecordKate, Msg1, OldTimestamp, ExpirationTime),
```

Comment:
Indentation :) this one will be hard as the function name is long and has many arguments. Still the indentation should be improved. Take a look at the indentation in lines 1036-1039, this is correct.